dist/emoji-en-US.json

```diff
@@ -4598,12 +4607,15 @@
     "rice_cracker",
     "food",
     "japanese",
-    "snack"
+    "snack",
+    "senbei"
```

Comment:
this one too

Reply:
"senbei" is just the japanese name for 🍘